test/run-unittest.d

```diff
@@ -107,5 +73,16 @@ int main(string[] args)
 		}
 	}
 
+	foreach(DirEntry script; dirEntries(curr_dir, (args.length > 1) ? args[1] : "*.script.d", SpanMode.shallow))
+	{
+		const min_frontend = script.name ~ ".min_frontend";
+		if (exists(min_frontend) && frontend.length && frontend < min_frontend.readText) continue;
+		log("Running " ~ script ~ "...");
+		if (spawnProcess([dub, script.name], ["DUB":dub, "DC":dc, "CURR_DIR":curr_dir]).wait)
```

Comment:
Usage of `DC` was only fixed in the latest version, meaning that anyone testing with an older compiler might get a different result. It won't show up in result, but it will for developers.